src/shader.cpp

```diff
@@ -37,6 +37,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "log.h"
 #include "gamedef.h"
 #include "client/tile.h"
+#include "cmake_config.h"
```

Comment:
This line breaks the android build as this file is only generated when you run cmake for the desktop build. I think this should be just `config.h`